.github/workflows/kubelinter-check.yml

```diff
@@ -0,0 +1,46 @@
+name: Kubelinter-check
+
+on:
+  push:
+    branches:
+    - master
+    paths-ignore: 
+    - docs/**
```

Comment:
I think maybe we don't need need to exclude any directories, it's possible that we could have yaml samples under `docs`